docs/questdb.md

```diff
@@ -0,0 +1,162 @@
+# TSBS Supplemental Guide: QuestDB
+
+QuestDB is a high-performance open-source time series database with SQL as a
+query language with time-oriented extensions. QuestDB implements PostgreSQL wire
+protocol, a REST API, and supports ingestion using InfluxDB line protocol.
+
+This guide explains how the data for TSBS is generated along with additional
+flags available when using the data importer (`tsbs_load_questdb`).
+**This should be read _after_ the main README.**
+
+## Data format
+
+Data generated by `tsbs_generate_data` is a "pseudo-CSV" format where each
```

Comment:
instead of psuedo-csv maybe we can just say influxdb line protocol

Reply:
Added in https://github.com/questdb/tsbs/commit/c5e93e7fa9cc67370f3a42ae3d40e3445bfa0b87 but it seems the PR is not updated